exchanges/bybit/bybit_ws_futures.go

```diff
@@ -0,0 +1,680 @@
+package bybit
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/gorilla/websocket"
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/common/crypto"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/stream"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/ticker"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/trade"
+	"github.com/thrasher-corp/gocryptotrader/log"
+)
+
+const (
+	wsFuturesPath = "realtime"
```

Comment:
this is unused.

Reply:
Yes, That's true. But it would be used when we try to add WS support for other assets like CMF, UMF etc.

Reply:
We can leave this for now, and I will resolve the other comment as well. If this will be included in another PR. Thanks for explaining that.